contents/handbook/brand/startups.md

```diff
@@ -4,77 +4,127 @@ sidebar: Handbook
 showTitle: true
 ---
 
-We run two special programs for early-stage teams. 
+We run two special programs for early-stage teams. The primary place for discussing both programs is the [#project-startups-and-yc](https://posthog.slack.com/archives/C088RSQKH2T) channel in Slack.
 
-1. [PostHog for Startups](/startups) is our program to support early stage companies.
-2. PostHog for Y Combinator is our program to support companies in the Y Combinator accelerator.
-
-Both programs give users some exclusive perks, as well as credit to use across any mix of PostHog products. 
+| Feature                     | Startups                      | Y Combinator                     |
+|----------------------------|-------------------------------|----------------------------------|
+| Eligibility                | <2 years old, <$5M raised     | Must be in YC (any batch)        |
+| Credit                     | $50,000 for 12 months         | $50k (recent batches) or $25k    |
+| BAA eligible?              | ❌ Cannot use credit for BAA  | ✅ Can use credit for BAA        |
+| Founder merch              | Welcome pack (max 4)          | Different welcome pack (max 4)   |
+| Community                  | —                             | Private YC founder Slack         |
+| Apply via…                 | [Startup page](/startups)     | [Secret YC page](https://app.posthog.com/startups/yc) |
 
 ## PostHog for Startups
-Any company that is <2 years old and has raised less than $5m in funding is eligible for PostHog for Startups.
 
-### What happens after companies apply
-1. Company applies by signing up to PostHog, completing onboarding, and then adding their billing details and applying on [our startups page](https://app.posthog.com/startups).
-2. If these criteria are fulfilled and the form gets submitted, we automatically apply a $50k Stripe _credit_ to their account that expires in 12 months.
-3. The user receives an automated email, via Zapier, which provides them with a confirmation about acceptance to the program (comes from [Scott](community/profiles/32112)).
-4. Shortly after, they get a separate email from Brilliant about claiming their merch (sent via Zapier, comes from [Scott](community/profiles/32112)).
-5. ~12h later, they get a welcome email with the details about the program and additional perks (sent via https://fly.customer.io/workspaces/127208/journeys/composer/actions/2185, comes from [Joe](community/profiles/29070)).
+Any company that is <2 years old and has raised less than $5M in funding is eligible to apply and claim the following:
+
+- $50,000 in PostHog credits (valid for 12 months)
+- Up to 4 unique welcome packs for founders (see [Welcome merch](#welcome-merch))
```

Comment:
As above - just 1 (at least in current implementation) for startups

Reply:
```suggestion
- One unique welcome pack for founders
```